client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/index.tsx

```diff
@@ -38,6 +41,13 @@ const mapApiError = ( error: any ) => {
 export const CredentialsForm: FC< CredentialsFormProps > = ( { onSubmit, onSkip } ) => {
 	const translate = useTranslate();
 
+	const [ passwordHidden, setPasswordHidden ] = useState( true );
+
+	const toggleVisibilityClasses = clsx( {
+		'site-migration-credentials__form-password__toggle': true,
+		'site-migration-credentials__form-password__toggle-visibility': ! passwordHidden,
```

Comment:
Is this class necessary? It doesn't look like it's being used anywhere.

Reply:
No, thanks for spotting, it was a leftover from a different approach I tried.